app/components/post_list/post/body/content/message_attachments/attachment_author.test.tsx

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import Preferences from '@constants/preferences';
+import {renderWithIntl} from '@test/intl-test-helper';
+
+import AttachmentAuthor from './attachment_author';
+
+describe('AttachmentAuthor', () => {
+    const baseProps = {
+        link: 'http://linktoatachment.com',
+        name: 'jhondoe',
```

Comment:
Typo?
```suggestion
        name: 'johndoe',
```

Reply:
LOL
<img width="794" alt="Screenshot 2023-09-29 at 15 57 48" src="https://github.com/mattermost/mattermost-mobile/assets/10082627/babfe9f3-b481-4665-b388-1a9aaef9aa80">

Reply:
So... not a typo then :rofl:

Reply:
I knew I remember pretty well my friends name "Jhon", actually like 10 to 15 "Jhons"